flaskdeid/medlp.py

```diff
@@ -9,16 +9,6 @@
 
 bp = Blueprint('medlp', __name__, url_prefix='/medlp')
```

Comment:
we're OK with leaving the URL endpoint as <whatever.org>/medlp/
?

Reply:
i think it's better to have a consistent convention for all our flask endpoints. i noticed in flaskblobnlp, endpoint is `<url>/endpoint/`, in flask deid, it is `<url>/endpoint` (without slash)

Reply:
honestly it should accept both. This is what got us into trouble with flaskblobnlp (throwing an error over bad endpoints when we passed it endpoint instead of endpoint/

Reply:
no...good point. comp_med?